xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -414,7 +422,11 @@ This method only creates a <xref:System.DateTime> representation of JSON strings
         
 -or-
         
-The entire UTF-8 encoded token value cannot be parsed to a <see cref="T:System.DateTime" /> value.</exception>
+The entire UTF-8 encoded token value cannot be parsed to a <see cref="T:System.DateTime" /> value.
+
+-or-
+
+The JSON token value is of an unsupported format. Only a subset of ISO 8601 formats are supported.</exception>
```

Comment:
```suggestion
The JSON token value is of an unsupported format.</exception>
```

I think the extra bit is redundant as the supported format is mentioned above. Here and below. cc @tdykstra